src/steps/os/windows.rs

```diff
@@ -128,12 +128,40 @@ fn upgrade_wsl_distribution(wsl: &Path, dist: &str, ctx: &ExecutionContext) -> R
     let topgrade = Command::new(wsl)
         .args(["-d", dist, "bash", "-lc", "which topgrade"])
         .output_checked_utf8()
-        .map_err(|_| SkipStep(String::from("Could not find Topgrade installed in WSL")))?;
+        .map_err(|_| SkipStep(String::from("Could not find Topgrade installed in WSL")))?
+        .stdout // The normal output from `which topgrade` appends a newline, so we trim it here.
+        .trim_end()
+        .to_owned();
 
     let mut command = ctx.run_type().execute(wsl);
+
+    // The `arg` method automatically quotes its arguments.
+    // This means we can't append additional arguments to `topgrade` in WSL
+    // by calling `arg` successively.
+    //
+    // For example:
+    //
+    // ```rust
+    // command
+    //  .args(["bash", "-c"])
+    //  .arg(format!("exec {topgrade}"));
+    // ```
+    //
+    // creates a command string like:
+    //  `bash -c 'exec /bin/topgrade'`
+    // and using `command.args(...).arg("topgrade").arg("-v")`
+    // on that again creates
+    //  `bash -c 'exec /bin/topgrade' -v`
+    // which means `-v` isn't passed to `topgrade`.
+    // ```
```

Comment:
1. It seems that the command string created by the following code
    
    ```rust
    command
        .args(["bash", "-c"])
        .arg(format!("exec {topgrade}"));
    ```

    should be: `wsl bash -c 'exec /bin/topgrade'`

    This should also be applied to `command.args(...).arg("topgrade").arg("-v")`

2. The ``` in line 156 seems to be unmatched, and thus should be removed?

Reply:
Thanks for the proofreading. :)

I pushed a new commit, copying some output from my machine for the sample outputs. Let me know if this looks good: https://github.com/topgrade-rs/topgrade/pull/556/commits/d0d2ac9d9cf2479ea70422dd886cbeefca33d4bd